Formula/e/envoy.rb

```diff
@@ -43,6 +43,10 @@ class Envoy < Formula
     cause "C++17 support and tcmalloc requirement"
   end
 
+  # Remove this patch when this is landed (or resolved) in a release:
+  # https://github.com/envoyproxy/envoy/pull/32148
```

Comment:
^ now merged into envoyproxy/envoy main

Reply:
It's been backported onto 1.29 (https://github.com/envoyproxy/envoy/commit/2cc41dcd321dab2356f29cd5902db65c97675291), so you should probably use that commit so the patch doesn't have to be copied to this repo

Reply:
Ah! I didn't realize. Thank you, will update it!